shared-ui/components/footer/Footer.tsx

```diff
@@ -21,6 +21,7 @@ import Facebook from '../../images/social-icons/facebook.svg';
 import Medium from '../../images/social-icons/medium.svg';
 import Twitter from '../../images/social-icons/twitter.svg';
 import PrimaryButton from '../primary-button/PrimaryButton';
+import MailinglistSignup from '../../../main-site/components/mailinglist-signup/MailinglistSignup';
```

Comment:
You need to update this to be
`import MailingListSignup from '../../../main-site/components/mailing-list-signup/MailingListSignup';`
and below when you render the component - this is failling your build